src/site/content/en/angular/route-preloading/index.md

```diff
@@ -0,0 +1,126 @@
+
+The sample app has several lazy-loaded routes. To preload all of them using the `PreloadAllModules` strategy—which is built into Angular—specify it as the value for the `preloadingStrategy` property in the router configuration:
+
+```js
+import { RouterModule, PreloadAllModules } from '@angular/router';
+// ...
+
+RouterModule.forRoot([
+  ...
+], {
+  preloadingStrategy: PreloadAllModules
+})
+// ...
+```
+
+Now let's serve the application and look at the **Network** panel in Chrome DevTools:
+
+{% Instruction 'devtools-network', 'ol' %}
+
+You should see that the router downloaded `nyan-nyan-module.js` and `about-about-module.js` in the background when you opened the application:
+
+<figure class="w-figure w-figure--fullbleed">
+<video controls loop muted poster="https://storage.googleapis.com/web-dev-angular/preloading/poster.png">
+  <source src="https://storage.googleapis.com/web-dev-angular/preloading/preload-all.webm" type="video/webm; codecs=vp8">
+  <source src="https://storage.googleapis.com/web-dev-angular/preloading/preload-all.mp4" type="video/mp4; codecs=h264">
+</video>
+ <figcaption class="w-figcaption w-figcaption--fullbleed">
+    Fig. 1 — PreloadAll Angular strategy
+  </figcaption>
+</figure>
+
+The router also registered the modules' route declarations so that when you navigate to a URL associated with any of the preloaded modules, the transition is instantaneous.
+
+
+### Using the Quicklink preloading strategy
+
+`PreloadAllModules` is useful in a lot of cases. When you have dozens of modules, however, its aggressive preloading can really increase network usage. Also, since the router needs to register the routes in all the preloaded modules, it can cause intensive computations in the UI thread and lead to sluggish user experience.
+
+The [quicklink](https://github.com/GoogleChromeLabs/quicklink) library provides a better strategy for larger apps. It uses the [IntersectionObserver](https://developers.google.com/web/updates/2019/02/intersectionobserver-v2) API to preload only modules associated with links that are currently visible on the page.
+
+You can add quicklink to an Angular app by using the [ngx-quicklink](https://www.npmjs.com/package/ngx-quicklink) package. To use `ngx-quicklink` you first have to install it from npm:
+
+```
+npm install --save quicklink
+```
+
+Once it's available in your project, you can use the `QuicklinkStrategy` by specifying the router's `preloadingStrategy` and importing the `QuicklinkModule`:
+
+```js
+import {QuicklinkStrategy, QuicklinkModule} from 'ngx-quicklink';
+...
+
+@NgModule({
+  ...
+  imports: [
+    ...
+    QuicklinkModule,
+    RouterModule.forRoot([...], {
+      preloadingStrategy: QuicklinkStrategy
+    })
+  ],
+  ...
+})
+export class AppModule {}
+```
+
+Now when you open the application again, you'll notice that the router only preloads `nyan-nyan-module.js` since the button in the center of the page has a router link to it:
```

Comment:
```suggestion
Now when you open the application again, you'll notice that the router only preloads `nyan-nyan-module.js` since the button in the center of the page has a router link to it. And when you open the side navigation, you'll notice that the router then preloads the "About" route:
```
Since there's only one video, these sentences should be in a single paragraph.